cdap-watchdog/src/main/java/co/cask/cdap/logging/gateway/handlers/LogHandler.java

```diff
@@ -92,18 +95,21 @@ public void getRunIdLogs(HttpRequest request, HttpResponder responder, @PathPara
                            @QueryParam("stop") @DefaultValue("-1") long toTimeSecsParam,
                            @QueryParam("escape") @DefaultValue("true") boolean escape,
                            @QueryParam("filter") @DefaultValue("") String filterStr,
-                           @QueryParam("format") @DefaultValue("text") String format) {
+                           @QueryParam("format") @DefaultValue("text") String format,
+                           @QueryParam("suppress") @DefaultValue("") List<String> suppress) {
```

Comment:
Has this been tested on SDK? Making sure that the queryparams supports List. Not sure if we already did support this.